tests/page-objects/contacts/contacts.wdio.page.js

```diff
@@ -26,6 +29,8 @@ const externalIdField = (place) => $(`[name="/data/${place}/external_id"]`);
 const notes = (place) => $(`[name="/data/${place}/notes"]`);
 const writeNamePlace = (place) => $(`[name="/data/${place}/is_name_generated"][value="false"]`);
 const contactCard = () =>$('.card h2');
+const rhsPeopleListSelector = () => 
```

Comment:
Can you use this selector instead of a pretty complex xpath?    This will include people and places. We should be able to add a test-id that indicates whether it is a place or person if need be. I think configurable hierarchy might throw a wrench in here too but we don't have that test automated yet. 

`$$('.card.children:not(.tasks):not(.reports) h4 span');`

Reply:
Done